src/baldr/tilehierarchy.cc

```diff
@@ -16,6 +16,10 @@ std::map<uint8_t, TileLevel> TileHierarchy::levels_  = {
     midgard::Tiles<midgard::PointLL>{{{-180, -90}, {180, 90}}, 4, static_cast<unsigned short>(kBinsDim)}}}
 };
 
+//Should we make a class lower than service other for transit?
+TileLevel TileHierarchy::transit_level_ = {2, stringToRoadClass.find("ServiceOther")->second, "transit",
```

Comment:
we may want to add this to other functions that get access to the levels but need to be careful of side effects due to transit being a special level